### PR TITLE
Fix backend container issues with webpack

### LINKED ISSUE #432: Docker compose module not found (docker and docker.wsgi)
**Describe the bug**
Getting a `ModuleNotFoundError: docker` error when trying to run `docker-compose up -d` on the backend.

I tried adding `pip install docker` to the Dockerfile but then I just get:
```
ModuleNotFoundError: No module named 'docker.wsgi'
```
**To Reproduce**
1. Install following README instructions
2. Run docker-compose up -d
3. Look at the logs of backend application

**Expected behavior**
Should run on localhost:8000 like it does when running outside of docker

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac OS X
 - Browser [e.g. chrome, safari] Chrome

### LINKED ISSUE #440: Error: EISDIR: illegal operation on a directory, open 'webpack-stats.json'
**Describe the bug**
webpack-stats.json appears to be a directory, but something is attempting to open it as a file.
```
frontend_1  | Error: EISDIR: illegal operation on a directory, open 'webpack-stats.json'
frontend_1  |     at Object.openSync (fs.js:457:3)
frontend_1  |     at Object.writeFileSync (fs.js:1298:35)
frontend_1  |     at Plugin.writeOutput (/app/node_modules/webpack-bundle-tracker/lib/index.js:124:6)
frontend_1  |     at compile (/app/node_modules/webpack-bundle-tracker/lib/index.js:58:10)
frontend_1  |     at SyncHook.eval (eval at create (/app/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:5:1)
frontend_1  |     at SyncHook.lazyCompileHook (/app/node_modules/tapable/lib/Hook.js:154:20)
frontend_1  |     at /app/node_modules/webpack/lib/Compiler.js:665:23
frontend_1  |     at AsyncSeriesHook.eval [as callAsync] (eval at create (/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)
frontend_1  |     at AsyncSeriesHook.lazyCompileHook (/app/node_modules/tapable/lib/Hook.js:154:20)
frontend_1  |     at Compiler.compile (/app/node_modules/webpack/lib/Compiler.js:662:28) {
frontend_1  |   errno: -21,
frontend_1  |   syscall: 'open',
frontend_1  |   code: 'EISDIR',
frontend_1  |   path: 'webpack-stats.json'
frontend_1  | }
```

In the backend:
```
backend_1   | Internal Server Error: /
backend_1   | Traceback (most recent call last):
backend_1   |   File "/usr/local/lib/python3.6/site-packages/webpack_loader/loader.py", line 26, in _load_assets
backend_1   |     with open(self.config['STATS_FILE'], encoding="utf-8") as f:
backend_1   | IsADirectoryError: [Errno 21] Is a directory: '/home/user/app/../webpack-stats.json'
backend_1   | 
...
```

**To Reproduce**
1. Install following README instructions
2. Run docker-compose up
3. Look at the logs

**Expected behavior**
There should be no errors when following the setup instructions without any modification. 

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina
 - Browser [e.g. chrome, safari] Chrome

### PR DESCRIPTION
**Pull request by jhgv**
## Description
The backend container was not having access to the webpack-stats.json file.
This PR  fixes #432 and closes #440

## Steps to reproduce
- Run the boilerplate locally with and without docker

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires dependencies updates.
- [ ] I have updated the dependencies accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If you made changes to this boilerplate and want to test them, do as following:
 - Run `git archive -o boilerplate.zip HEAD` to create the template zip file
 - Run the following:
   ```bash
-  cd .. && django-admin startproject theprojectname --extension py,yml,json --name Procfile,README.md,.env.example --template=django-react-boilerplate/boilerplate.zip
+  cd .. && django-admin startproject theprojectname --extension py,yml,json --name Procfile,README.md,.env.example,Dockerfile --template=django-react-boilerplate/boilerplate.zip
   ```
 - A new folder called `theprojectname` will be created and now you can test your changes
```
Comment (vanessa) [inline CONTRIBUTING.md]: Add a step here to test the changes both with and without Docker

**Comment by vanessa** [comment]
@jhgv Errored when I tried to run `docker-compose up`:
```
 ---> 7d0ee93663d0
Step 10/13 : COPY webpack-stats.json/ /home/user
ERROR: Service 'backend' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder683044911/webpack-stats.json: no such file or directory
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (jhgv) [reply CONTRIBUTING.md]: done

File: docker-compose.yml [hunk 2]
```diff
@@ -30,14 +30,12 @@ services:
...
       - ./backend/:/home/user/app/
-      - ./webpack-stats.json:/home/user/webpack-stats.json
-      - ./jquery-webpack-stats.json:/home/user/jquery-webpack-stats.json
     env_file: backend/.env
     depends_on:
       - db
       - broker
       - result
-
+      - frontend
```
[the remark this answers is not in the document]
Comment (jhgv) [reply docker-compose.yml]: done

**Comment by jhgv** [comment]
@vanessa It should be fine now

**Comment by vanessa** [comment]
Nice! I'll review it by the end of this week.

**Pull request merged**